src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/permissions/PermissionsClient.scala

```diff
@@ -126,6 +126,25 @@ trait PermissionsClient {
 
   }
 
+  def listOwners(agoraEntity: AgoraEntity): Seq[String] = {
+    val permissionsQuery = for {
+      entity <- entities if entity.alias === alias(agoraEntity)
+      _permissions <- permissions if _permissions.entityID === entity.id // && _permissions.roles===Manage
```

Comment:
I'm not sure if I should be checking to see if you have manage/owner permissions within the sql query itself instead of filtering the future/result like what I'm currently doing